docs/includes/yeoman-generator-recommendation.md

```diff
@@ -0,0 +1,6 @@
+> [!TIP]
+> Although it's possible to create Office Add-ins using Visual Studio, using the Yeoman generator provides a better developer experience in some notable ways.
+> 
+> * The Yeoman generator provides a wider range of options for project types, frameworks, and languages than Visual Studio does for Office Add-in projects.
+> 
+> * Project templates in the Yeoman generator are updated more frequently than project templates in Visual Studio. This means that as new types of Office Add-ins become available in the future, they're likely to be supported by the Yeoman generator before they're supported by Visual Studio. Also, as recommended programming patterns continue to evolve, the Yeoman generator project templates will often reflect those new patterns long before they're available in the Visual Studio project templates.
```

Comment:
I'm not sure this section needs to answer this, but I can see users asking "why does VS update slower?"

Reply:
Yep, I agree that some folks may wonder about that. Not sure how to provide that info here in an objective but acceptable way though.

Reply:
Fair. I guess we could be somewhat honest and say it takes Visual Studio updates less frequently. That would at least shift the spin from "VS doesn't update" to "yeoman updates faster".